### ISSUE TITLE
Documentation - How to use values from previous step?

### ISSUE THREAD

**Issue by patrickkusebauch** [contributte/forms-wizard#10]
Hello,
 have used the template for `latte` used in the quickstart.
```html
<div n:wizard="wizard">
    <ul n:if="!$wizard->isSuccess()">
        <li n:foreach="$wizard->steps as $step" n:class="$wizard->isDisabled($step) ? disabled, $wizard->isActive($step) ? active">
            <a n:tag-if="$wizard->useLink($step)" n:href="changeStep! $step">{$step}</a>
        </li>
    </ul>

    {step 1}
        {control $form}
    {/step}

    {step 2}
        {control $form}
    {/step}

    {step success}
        Registration was successful
    {/step}
</div>
```

However, whenever I reference a value from a previous step in the Component like this:
```php
            protected function createStep2(): Form
            {
                $values = $this->getValues();
                $form   = $this->createForm();

                if($values->seasonDesc === 'personal') {
                    //conditional form based on values from the first step
                   }
            }
```

I get `Notice: Undefined property: Nette\Utils\ArrayHash::{$nameOfProperty}` even though I know the property should exist since it was created in the previous step.

This is the whole point of a wizard for me, that I can change the next step's form base on the values from the previous step. So is there any canonical way to do this other than always calling `$values->offsetExists('seasonDesc')`?

**Comment by MartkCz** [comment]
I'm working on reworking the wizard to fix #10 #9

**Comment by MartkCz** [comment]
#11 

```php
bdump($this->getRawValues()[1]['seasonDesc']);
```

Please test PR
